src/test/java/com/mael/payroll/transactions/AddCommissionedEmployeeTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mael.payroll.transactions;
+
+import com.mael.payroll.*;
+import com.mael.payroll.paymentSchedules.CommissionedSchedule;
+import com.mael.payroll.paymentTypes.CommissionedPayment;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AddCommissionedEmployeeTest {
+
+    private Employee employee;
+
+    @Before
+    public void setUp() {
+        PayrollDBFacade payrollDB               = new PayrollDBFacade();
+        AddEmployeeTransaction addCommissionedE = new AddCommissionedEmployee(1, "Squiddo", "FishBowl", 1000.0, 2.0);
```

Comment:
I don't see any verification of the values `"FishBowl"`, `1000.0` or `2.0`.
Do they mean anything? Are they needed? If not, why pass them?